packages/xo-server/src/xo-mixins/metadata-backups.js

```diff
@@ -347,22 +348,24 @@ export default class metadataBackup {
 
         let outputStream
         try {
-          await Promise.all([
+          await all.call([
```

Comment:
Why are you using this instead of `Promise.all`?

Reply:
We need to wait for all promises to be fulfilled before deleting the backup. Please see  https://github.com/vatesfr/xen-orchestra/pull/4666/files#diff-fe664de5297cdcda5f09b752eeab1fd6R401

Reply:
I don't see the point of this diff, I know what I said before.

Are you sure this is the behavior of `promise-toolbox/all`?

Reply:
https://github.com/JsCommunity/promise-toolbox/blob/master/src/all.js#L49

Reply:
1. a comment is not the same as the documentation, but let's go with it
2. no, you did not read the comment correctly.

Reply:
Do you mean the promise returned by `all` never reject?